exercises/hamming/hamming_test.exs

```diff
@@ -8,34 +8,39 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule DNATest do
   use ExUnit.Case
 
-  # @tag :pending
+#   @tag :pending
+  test "hamming distance for strands of the same length are ok" do
+    assert {:ok, _} = DNA.hamming_distance('', '')
+  end
```

Comment:
this test seems really redundant actually. we'll be checking for `{:ok, _}` in about every test already